### PR TITLE
Add a cleanup_test_data management command to remove old test data from dev and stage

### PR DESCRIPTION
**Pull request by bobsilverberg**
This is to address https://github.com/mozilla/moztrap-tests/issues/173, which came about as a result of https://bugzilla.mozilla.org/show_bug.cgi?id=1212069

@camd r?

Note that if this is accepted and merged we will need to set up a cron job to run this periodically.

### CODE REVIEW

File: moztrap/model/core/management/commands/cleanup_test_data.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                    action='store_true',
+                    dest='permanent',
+                    default=True,
+                    help='Permanently delete records?'),)
+
+    def handle(self, *args, **options):
+        for model in (core_models.Product,
+                      env_models.Category,
+                      env_models.Element,
+                      env_models.Profile):
```
Comment (camd) [inline moztrap/model/core/management/commands/cleanup_test_data.py]: I think we need ``env_models.Environment`` in this list as well.  The order may matter, but I'm not certain.  Might need to be ``Profile``, ``Environment``, ``Element``, ``Category``.  But you should be sure to test it on your local machine, of course.  :)

Comment (camd) [inline moztrap/model/core/management/commands/cleanup_test_data.py]: Hrm...  I could be wrong.  Perhaps these actually cascade deletes for the envs?  In which case, as long as they're all gone after you run it, doesn't matter.  So feel free to push back on that.  :)

Comment (bobsilverberg) [inline moztrap/model/core/management/commands/cleanup_test_data.py]: Yes, @camd, they cascade, so all we need to delete everything is what is currently in the code. I tested it a number of times on my machine and this does the trick.

**Comment by camd** [comment]
Thanks Bob!
Pushing to stage now...

**Pull request merged**